file_store/src/mobile_session.rs

```diff
@@ -190,6 +191,7 @@ impl MsgDecode for DataTransferSessionReq {
 impl TryFrom<DataTransferSessionReqV1> for DataTransferSessionReq {
     type Error = Error;
 
+    #[allow(deprecated)]
```

Comment:
Instead of doing this, don't set the deprecated field and do ..Default::default() at the end of the struct

Reply:
Addressed in d368f75